src/controlCenter/widgets/mpris/mpris_player.vala

```diff
@@ -96,6 +96,7 @@ namespace SwayNotificationCenter.Widgets.Mpris {
                     update_buttons (source.media_player.metadata);
                 });
             });
+            album_art.set_pixel_size(mpris_config.image_size);
```

Comment:
Oh wait, seems like the linter isn't happy about this line. Could you fix that?

Reply:
Done. There are a lot of linter changes showing up, so I missed this one (I did not want to add unrelated line changes n this PR).
Maybe it would be nice to lint the whole project and do a single commit with that.